drivers/python/tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+skipsdist = True
+envlist = py27, py34
+
+[base]
+commands =
+    pip install -e . -r requirements-test.txt
+
+[testenv:py27]
+pip_pre=False
+commands =
+    {[base]commands}
+    {toxinidir}/../../test/rql_test/test-runner -i py2.7 {posargs}
+
+[testenv:py34]
+pip_pre=False
+commands =
+    {[base]commands}
+    {toxinidir}/../../test/rql_test/test-runner -i py3 {posargs}
```

Comment:
This should be `-i py3.4` if you are specifying Python3.4. Otherwise you will get Python3.0 if it is installed. With `twisted` not being supported on Python3.0 (grumble, grumble), that would defeat some of the purpose.

Also, why not include versions for 2.6, 3.0, 3.1, 3.2, and 3.3? That way anyone who wants to can use if for any version.